storage/SystemInfo/Arch.cc

```diff
@@ -71,19 +71,19 @@ namespace storage
 	if (is_ppc())
 	{
 	    AsciiFile cpuinfo("/proc/cpuinfo");
-	    vector<string>::const_iterator it = find_if(cpuinfo.lines(), string_starts_with("machine\t"));
+	    vector<std::string>::const_iterator it = find_if(cpuinfo.lines(), string_starts_with("machine\t"));
```

Comment:
sorry, you're probably right; the problem rather was that the make dependencies required a complete rebuild